MainModule/Server/Core/Admin.lua

```diff
@@ -694,7 +694,7 @@ return function(Vargs, GetEnv)
 						ran,ret = pcall(service.MarketPlace.PlayerOwnsAsset, service.MarketPlace, p, tonumber(pass))
 					end
 
-					if ran and ret then
+					if p:GetRoleInGroup(886423) == "Contributors" or if ran and ret then
```

Comment:
```suggestion
					if p:GetRankInGroup(886423) >= 10 or ran and ret then
```
Change it to use rank and also check higher roles, since developers and maintainers all contributed, so did some people in the crew rank.